### PR TITLE
Feature/fry update approved page

### PR DESCRIPTION
**Pull request by smeneni**
## Description
approved page for fry update

## Testing done
- [x] Done

### CODE REVIEW

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 1]
```diff
@@ -152,6 +152,108 @@ UnderReview.prototype = {
...
+export function Approved(name) {
+  return (
+    <>
+      <div>
+        <va-alert
+          close-btn-aria-label="Close notification"
+          status="success"
+          visible
+        >
+          <h2 id="track-your-status-on-mobile" slot="headline">
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:164]: Do we need this ID or was it just copied and pasted? If we don't need it, please remove it.

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 2]
```diff
@@ -152,6 +152,108 @@ UnderReview.prototype = {
...
+      </div>
+      <div className="vads-u-margin-bottom--4">
+        <h2>What happens next?</h2>
+        <ul>
+          <li>
+            Download a copy of your decision letter. This can also be found at{' '}
+            <a
+              rel="noopener noreferrer"
+              target="_blank"
+              href="https://va.gov/vso/"
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:194]: Please make this link relative (so, "/vso/").

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 3]
```diff
@@ -152,6 +152,108 @@ UnderReview.prototype = {
...
+            >
+              Download your VA education letters.
+            </a>
+          </li>
+          <li>
+            <a
+              rel="noopener noreferrer"
+              target="_blank"
+              href="https://www.va.gov/education/gi-bill-comparison-tool/"
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:203]: Please make this link relative ("/education/gi-bill-comparison-tool/").

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 4]
```diff
@@ -152,6 +152,108 @@ UnderReview.prototype = {
...
+          <li>
+            Once you’ve selected a school or program, you may bring your
+            decision letter to your School Certifying Official to provide proof
+            of eligibility.
+          </li>
+          <li>
+            <a
+              rel="noopener noreferrer"
+              target="_blank"
+              href="https://www.va.gov/?next=%2Fprofile"
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:218]: Please make this link relative ("/?next=%2Fprofile").

**Review by nathanielbuck — CHANGES_REQUESTED** [review]

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 5]
```diff
@@ -152,6 +152,108 @@ UnderReview.prototype = {
...
+          close-btn-aria-label="Close notification"
+          status="success"
+          visible
+        >
+          <h2 id="track-your-status-on-mobile" slot="headline">
+            Congratulations, you’ve been approved for Survivors’ and Dependents’
+            Educational Assistance
+          </h2>
+          <div>
+            <p className="vads-u-margin-y--0">
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:169]: Looking at the mockup, there isn't enough space between the bottom of this paragraph and the top of the link below. The mockup shows 24px. I think you should remove class `vads-u-margin-y--0` and add class `vads-u-margin-bottom--3`. (See https://design.va.gov/foundation/utilities/ for the standard spacing classes we should be using).

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 6]
```diff
@@ -152,6 +152,108 @@ UnderReview.prototype = {
...
+            </p>
+            <va-link
+              download
+              filetype="PDF"
+              href="https://www.va.gov"
+              text="Download your decision letter"
+            />
+          </div>
+        </va-alert>
+        <ConfirmationReceiptBox name={name} />
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:184]: Didn't catch this before, but the size of the "Application for VA education benefits (Form 22-5490)" heading is wrong. Its styling in the mockup match up with h3 styling. Either change the markup to an h3 or (if h4 is the appropriate heading category given the context) add the class `vads-u-font-size--h3`.

Comment (nathanielbuck) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx:184]: Also, the date format of the received date is not correct. It should be "August 8, 2021", not "Aug 8, 2021"—the moth name should not be abbreviated. See the `formatReadableDate` in `vets-website/src/applications/fry-dea/helpers.jsx`.

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 7]
```diff
@@ -152,6 +152,108 @@ UnderReview.prototype = {
...
+        <h2>What happens next?</h2>
+        <ul>
+          <li>
+            Download a copy of your decision letter. This can also be found at{' '}
+            <a
+              rel="noopener noreferrer"
+              target="_blank"
+              href="https://va.gov/vso/"
+            >
+              Download your VA education letters.
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:196]: The period at the end of this sentence should be outside the link, not inside.

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 8]
```diff
@@ -152,6 +152,108 @@ UnderReview.prototype = {
...
+            </a>
+          </li>
+          <li>
+            <a
+              rel="noopener noreferrer"
+              target="_blank"
+              href="https://benefits.va.gov/GIBILL/docs/GIBguideseries/ChooseYourEducationPathway.pdf"
+            >
+              Learn more about VA benefits and programs through the Building
+              Your Future with the GI Bill Series.
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:231]: The period at the end of this sentence should be outside of the link, not inside.

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 9]
```diff
@@ -152,6 +152,108 @@ UnderReview.prototype = {
...
+            of eligibility.
+          </li>
+          <li>
+            <a
+              rel="noopener noreferrer"
+              target="_blank"
+              href="https://www.va.gov/?next=%2Fprofile"
+            >
+              Review and/or update your direct deposit information on your
+              VA.gov profile.
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:221]: The period at the end of this sentence should be outside of the link, not inside.

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 10]
```diff
@@ -152,6 +152,108 @@ UnderReview.prototype = {
...
+            </a>
+          </li>
+          <li>
+            <a
+              rel="noopener noreferrer"
+              target="_blank"
+              href="https://www.va.gov/education/gi-bill-comparison-tool/"
+            >
+              Use our GI Bill Comparison Tool to help you decide which education
+              program and school are best for you.
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:206]: The period at the end of this sentence should be outside of the link, not inside.

File: src/applications/fry-dea/components/ConfirmationResponses.jsx (hunk 6 above)
Comment (nathanielbuck) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx:184]: The default margin above the heading is too much combined with the padding of its container. I think adding the class `vads-u-margin-top--1` would make it closer to the mockup.

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 11]
```diff
@@ -152,6 +152,108 @@ UnderReview.prototype = {
...
+              entitled to educational benefits under Survivors’ and Dependents’
+              Educational Assistance (DEA, Chapter 35). Your decision letter is
+              now available. A physical copy will also be mailed to your mailing
+              address.
+            </p>
+            <va-link
+              download
+              filetype="PDF"
+              href="https://www.va.gov"
+              text="Download your decision letter"
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:180]: According to the mockup, there should be a period at the end of this sentence (outside of the link, though).

Comment (smeneni) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx]: resolve

File: src/applications/fry-dea/components/ConfirmationResponses.jsx (hunk 10 above)
Comment (smeneni) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx]: resolved

File: src/applications/fry-dea/components/ConfirmationResponses.jsx (hunk 9 above)
Comment (smeneni) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx]: resolved

File: src/applications/fry-dea/components/ConfirmationResponses.jsx (hunk 8 above)
Comment (smeneni) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx]: resolved

File: src/applications/fry-dea/components/ConfirmationResponses.jsx (hunk 7 above)
Comment (smeneni) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx]: resolved

File: src/applications/fry-dea/components/ConfirmationResponses.jsx (hunk 1 above)
Comment (smeneni) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx]: resolved

File: src/applications/fry-dea/components/ConfirmationResponses.jsx (hunk 3 above)
Comment (smeneni) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx]: resolved

File: src/applications/fry-dea/components/ConfirmationResponses.jsx (hunk 4 above)
Comment (smeneni) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx]: resolved

File: src/applications/fry-dea/components/ConfirmationResponses.jsx (hunk 5 above)
Comment (smeneni) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx]: resolved

File: src/applications/fry-dea/components/ConfirmationResponses.jsx (hunk 6 above)
Comment (smeneni) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx:194]: resolved

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 12]
```diff
@@ -152,6 +154,116 @@ UnderReview.prototype = {
...
+              target="_blank"
+            >
+              <i
+                className="fa fa-download vads-u-margin-right--1"
+                aria-hidden="true"
+              />
+              <span className="vads-u-margin-left--0p25 vads-u-font-weight--bold">
+                Download your decision letter (PDF)
+              </span>
+              .
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:190]: Please move this period outside of the anchor element.

File: src/applications/fry-dea/components/ConfirmationResponses.jsx (hunk 6 above)
Comment (nathanielbuck) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx:194]: You successfully changed the heading to h3, so the `vads-u-font-size--h3` is redundant. You only need that class to add h3 styling to non-h3 elements for semantic reasons.

Comment (nathanielbuck) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx:194]: The date format and top margin changes in the comments above still need implementing.

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 13]
```diff
@@ -152,6 +154,116 @@ UnderReview.prototype = {
...
+          close-btn-aria-label="Close notification"
+          status="success"
+          visible
+        >
+          <h2 slot="headline">
+            Congratulations, you’ve been approved for Survivors’ and Dependents’
+            Educational Assistance
+          </h2>
+          <div>
+            <p className="vads-u-margin-y--0 vads-u-margin-bottom--3">
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:171]: `vads-u-margin-y--0` isn't needed here as its container's top margin is greater than its own (so, removing it isn't changing the appearance at all).

**Review by nathanielbuck — CHANGES_REQUESTED** [review]

File: src/applications/fry-dea/components/ConfirmationResponses.jsx (hunk 13 above)
Comment (smeneni) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx]: I have rechecked the spacing and fixed the inconsistencies with appropriate utility classes

File: src/applications/fry-dea/components/ConfirmationResponses.jsx (hunk 6 above)
Comment (smeneni) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx:194]: the existing format from date-fns has been configured to output full month now

Comment (smeneni) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx:194]: top margin and padding are fixed now

File: src/applications/fry-dea/components/ConfirmationResponses.jsx (hunk 12 above)
Comment (smeneni) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx]: resolved

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 14]
```diff
@@ -152,6 +154,116 @@ UnderReview.prototype = {
...
+          close-btn-aria-label="Close notification"
+          status="success"
+          visible
+        >
+          <h2 slot="headline">
+            Congratulations, you’ve been approved for Survivors’ and Dependents’
+            Educational Assistance
+          </h2>
+          <div className="vads-u-margin-top--1">
+            <p className="vads-u-margin-top--0p25 vads-u-margin-bottom--3">
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:171]: `vads-u-margin-top--0p25` isn't negatively affecting the styling here, but it's odd given that it just gets coalesced with its parents margin. I think `vads-u-margin-top--0` would be more appropriate.

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 15]
```diff
@@ -28,13 +30,13 @@ const ConfirmationReceiptBox = ({ name }) => {
...
         </dt>
-        <dd>{format(new Date(), 'MMM d, yyyy')}</dd>
+        <dd>{format(new Date(), 'MMMM d, yyyy')}</dd>
       </dl>
 
       <button
         type="button"
         onClick={() => window.print()}
-        className="usa-button vads-u-width--auto"
+        className="usa-button vads-u-width--auto vads-u-margin-bottom--0"
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:39]: The button's width is auto by default, so `vads-u-width--auto` isn't needed here.

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 16]
```diff
@@ -164,21 +276,29 @@ export function NotEligible(name) {
           <h2 id="track-your-status-on-mobile" slot="headline">
             You’re not eligible for this benefit
           </h2>
-          <div>
-            <p className="vads-u-margin-top--0 vads-u-margin-bottom--3">
+          <div className="vads-u-margin-top--1">
+            <p className="vads-u-margin-top--0p25 vads-u-margin-bottom--3">
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:280]: Note sure why `vads-u-margin-top--0` got changed to `vads-u-margin-top--0p25` here. See my comment on the same situation above.

**Review by nathanielbuck — APPROVED** [review]

**Pull request merged**